apps/bandwagon/tests/test_models.py

```diff
@@ -138,6 +138,17 @@ def test_publishable_by(self):
         CollectionUser(collection=c, user=self.user).save()
         eq_(c.publishable_by(self.user), True)
 
+    def test_manager_publishable_by(self):
+        c1 = Collection.objects.create(author=self.user, name='B')
+        c2 = Collection.objects.create(author=self.user, name='A')
+        c3 = Collection.objects.create(author=self.other, name='D')
+        Collection.objects.create(author=self.other, name='C')
+        CollectionUser(collection=c1, user=self.user).save()
+        CollectionUser(collection=c2, user=self.other).save()
+        CollectionUser(collection=c3, user=self.user).save()
```

Comment:
For good measure, we should also have a `CollectionUser` for the "other" user here, just to make sure that even if there's one, it still doesn't return it in the publishable list.